### DISCUSSION TITLE
APC KEY 25 MK2 IN PLAYTIME BUT NOT IN REALEARN

### DISCUSSION THREAD

**Discussion by radjoulbalma** [helgoboss/helgobox#1283] [Q&A]
In the controler list of realearn you mention the apc key 25 but not the mk2 one. But the latter in mentioned as usable for playtime 2.
So is  the apc key 25 mk2 usable with realearn ? (I imagine yes...)

Thank you

**Comment by helgoboss** [comment]
Yes, definitely. You can use it with ReaLearn. Just use the corresponding controller preset.

Most MIDI controller can be used with ReaLearn, by the way, even if they are not on the list.

But yeah, I should mention this one.
